### PR TITLE
Feat: create attendances download route

### PR DESCRIPTION
**Pull request by Gonzagadavid**
# Description 

- create route dowload to attendances;
- add to csv
- return buffer

# Related issue

- part of issue [Create download routes](https://trello.com/c/bJQceH1c/29-create-download-routes)

### CODE REVIEW

File: src/presentation/api/attendances.ts [hunk 1]
```diff
@@ -45,6 +51,49 @@ export class AttendanceAPI extends Controller {
     this.volunteerRepository = volunteerRepository;
   }
 
+  /**
+   * Get download all attendances from a specified date (the format of the date parameter is: yyyy-mm-dd)
```
Comment (JVSRamos) [inline src/presentation/api/attendances.ts:55]:
```suggestion
   * Get download all attendances from a specified date (the format of the date parameter is: yyyy-mm-dd)
   * OBS: This route returns the data as a stream with attachment headers
```

**Review by JVSRamos — CHANGES_REQUESTED** [review]
Well Done! I just requested a change on the route documentation, specifying that it returns a stream as an attachment, as tsoa does not generate it automatically in this way.

File: src/presentation/api/attendances.ts (hunk 1 above)
Comment (Gonzagadavid) [reply src/presentation/api/attendances.ts:55]: done!

**Review by JVSRamos — APPROVED** [review]
Nice!

**Pull request merged**